README.md

```diff
@@ -255,8 +255,97 @@ created in Step3 is listed
 
 
 ### 4. Develop Watson Knowledge Studio model
-Detailed steps to create a model in WKS is provided [here](https://github.ibm.com/IBMCode/howtos/tree/master/extracting-personal-data-from-unstructured-text-using-watson-knowledge-studio). 
-This is a how-to guide. Follow these steps to create a custom model.
+## 4.1 Import Artifacts
+In github repository, navigate to WKS folder. Download the files named “Documents.zip” and
+“TypeSystems.json” to your local filesystem.
+
+
+## 4.2 Create Project
+Login to the WKS.
```

Comment:
This should be a link to the WKS login.

Reply:
The WKS login link is unique to each user and that's why link is not provided. To add clarity, I have added instructions to note down the WKS login URL in prerequisites steps and use the same login URL to login to WKS